tasksCalendar/view.js

```diff
@@ -141,16 +146,22 @@ function getTasks(date) {
 function setTask(obj, type) {
 	var noteColor = getColor(obj);
 	var noteIcon = getIcon(obj);
+	var textColor = getTextColor(obj);
 	var taskText = obj.text.replace("'", "&apos;");
 	var taskPath = obj.link.path.replace("'", "&apos;");
 	var taskSubpath = obj.header.subpath;
 	var taskLine = taskSubpath ? taskPath+"#"+taskSubpath : taskPath;
 	var style = "";
+	console.log(textColor);
 	// if (noteColor) { style = "color:" + noteColor + ";background:" + noteColor + transparency };
-	if (noteColor) {
-		style = "--task-background:"+noteColor+"33;--task-color:"+noteColor;
```

Comment:
First if statement is not necessary because it's catched with the second and third if statement and this if doesn't end with a curly bracket before the second if statement begins